nilearn/plotting/img_plotting.py

```diff
@@ -76,6 +79,51 @@ def _get_plot_stat_map_params(stat_map_img, vmax, symmetric_cbar, kwargs, force_
         cbar_vmin, cbar_vmax = None, None
     return cbar_vmin, cbar_vmax, vmin, vmax
 
+
+def check_threshold(threshold, data, percentile_calculate, name=None):
```

Comment:
This function is in `displays.py` and in `img_plotting.py`. You only need one version ;).